src/Files.App/Helpers/Application/AppLifecycleHelper.cs

```diff
@@ -26,6 +27,39 @@ namespace Files.App.Helpers
 	/// </summary>
 	public static class AppLifecycleHelper
 	{
+		/// <summary>
+		/// Gets the value that provides application environment or branch name.
+		/// </summary>
+		public static AppEnvironment AppEnvironment { get; } =
+#if STORE
+			AppEnvironment.Store;
+#elif PREVIEW
+			AppEnvironment.Preview;
+#elif STABLE
+			AppEnvironment.Stable;
+#else
+			AppEnvironment.Dev;
+#endif
+
+		/// <summary>
+		/// Gets application package version.
+		/// </summary>
+		public static Version AppVersion { get; } = new(
+			Package.Current.Id.Version.Major,
+			Package.Current.Id.Version.Minor,
+			Package.Current.Id.Version.Build,
+			Package.Current.Id.Version.Revision);
+
+		/// <summary>
+		/// Gets application icon relative path.
+		/// </summary>
+		public static string AppIconPath { get; } = AppEnvironment switch
```

Comment:
What do you think about combining this with `Package.Current.InstalledLocation.Path`?

Reply:
Fair enough, thanks!

Reply:
Done